Bottom: 9040f434242f27ea6eabf31f345b90a5c2b00727
Top:    668d2571b33cd79e8fd25cbbc272826972bef8b5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-06-03 23:35:41 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index d7033e1..1237607 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -323,13 +323,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
